versioned_docs/version-2.0/batch-processing/quiz.md

```diff
@@ -73,11 +75,7 @@ via broadcast join (small join tables) OR partition your data such that data for
 <details><summary>What’s a good rule of thumb for partitioning?</summary>
 Partition on columns that you would typically do a filter/groupBy 
 
-A partition should ideally contain anywhere between 256MB - 2GB of data. Too many small partitions (each containing kilobytes means you have lots of small files - that’s bad!)
+A partition should ideally contain anywhere between 256MB - 1GB of data. Too many small partitions (each containing kilobytes means you have lots of small files - that’s bad!)
```

Comment:
Learning material says 256MB - 1GB.

Reply:
I think 1GB is probably right here so far, 2GB seems like a typo. I've asked Daniel for some links for where he's found that information since an initial search didn't find those recs by those exact words. Anyway, 1024MB is about 1GB anyway. Let's get this PR out first and then we can revisit the course material in a separate issue. Thanks for the investigation! 🎉

Reply:
https://github.com/data-derp/data-derp.github.io/issues/102